--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabUiFeatureUtilities.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabUiFeatureUtilities.java
@@ -40,6 +40,7 @@ public class TabUiFeatureUtilities {
      * @return Whether the Tab Grid / Group UI is enabled.
      */
     public static boolean isGridTabSwitcherEnabled(Context context) {
+        if (true) return false;
         return ChromeFeatureList.sTabGridLayoutAndroid.isEnabled();
     }
